packages/react-components/src/organisms/UserCollaborationTable.tsx

```diff
@@ -174,13 +189,21 @@ const displayOutputsCount = (items: UserCollaborationMetric['teams']) => {
 
 type UserCollaborationTableProps = ComponentProps<typeof PageControls> & {
   data: UserCollaborationMetric[];
+  performance: PerformanceMetrics;
 };
 
 const UserCollaborationTable: React.FC<UserCollaborationTableProps> = ({
   data,
+  performance,
   ...pageControlProps
 }) => (
   <>
+    <CaptionCard>
+      <>
+        <CaptionItem label="Outputs Co-Authored" {...performance} />
```

Comment:
The caption item looks a bit different from the designs

UI
<img width="572" alt="Screenshot 2024-07-02 at 10 35 45" src="https://github.com/yldio/asap-hub/assets/25244556/91c3d37f-8db8-4739-8c15-4f47b33b1a23">

Designs
<img width="572" alt="Screenshot 2024-07-02 at 10 41 51" src="https://github.com/yldio/asap-hub/assets/25244556/3d1aa6ae-b3d4-4616-bfbf-5f3a55324476">